lib/common/models/wp_user/brute_forcable.rb

```diff
@@ -28,9 +28,15 @@ def brute_force(wordlist, options = {}, redirect_url = nil)
       queue_count  = 0
       found        = false
 
-      create_progress_bar(count_file_lines(wordlist)+1, options)
+      if wordlist == '-'
+        wordlist = $stdin.readlines
```

Comment:
this would read the wordlist completely into memory (which can be bad in case of a big wordlist)

I think we need to pass this info to `brute_force` which should then loop over stdin line by line (or the wordlist)